### DISCUSSION TITLE
Want a working video

### DISCUSSION THREAD

**Discussion by yilinalinalin** [HarveyBates/Open-JIP#11] [Q&A]
Hi Harvey:

So excited to see your work! Can I have a working video of it? So that I can better understand how to build the circuit and assemble the hardware. I'm going to use this to test the plant's chlorophyll, instead of the algae's.

Thanks!

**Comment by dkbastola** [comment]
Hi Harvey,
Echo what "yilinalinalin", I am interested in the study of both plants as well as algae. Thanks.
